src/components/Calculator.js

```diff
@@ -43,39 +21,117 @@ Output.defaultProps = {
+        <Output result={total} />
         <div className="keysGrid">
-          <Button className="calcBtn" value="AC" />
-          <Button className="calcBtn" value="+/-" />
-          <Button className="calcBtn" value="%" />
-          <Button className="calcBtn" value="+" />
-          <Button className="calcBtn" value="7" />
-          <Button className="calcBtn" value="8" />
-          <Button className="calcBtn" value="9" />
-          <Button className="calcBtn" value="x" />
-          <Button className="calcBtn" value="4" />
-          <Button className="calcBtn" value="5" />
-          <Button className="calcBtn" value="6" />
-          <Button className="calcBtn" value="-" />
-          <Button className="calcBtn" value="1" />
-          <Button className="calcBtn" value="2" />
-          <Button className="calcBtn" value="3" />
-          <Button className="calcBtn" value="+" />
-          <Button className="calcBtn" value="0" />
-          <Button className="calcBtn" value="." />
-          <Button value="=" />
+          <Button
+            value="AC"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="+/-"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="%"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="+"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="7"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="8"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="9"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="x"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="4"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="5"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="6"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="-"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="1"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="2"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="3"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="+"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="0"
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="."
+            handleClick={this.handleClick}
+          />
+          <Button
+            value="="
+            handleClick={this.handleClick}
+          />
         </div>
       </div>
```

Comment:
- [ ] Please the number key does not display when click it only displays when an operation sign is clicked. Kindly fix this

https://user-images.githubusercontent.com/55596970/167275990-6fb7160d-d784-4c41-9aae-e3edbd4f6cbc.mp4